templates/statefulset.yaml

```diff
@@ -209,6 +209,7 @@ spec:
 
       nodeSelector:
         {{ .Values.nodeSelector.nodeSelectorKey }}: {{ .Values.nodeSelector.nodeSelectorValue }}
+        {{ .Values.nodeSelector.nodeSelectorAllow }}: {{ .Values.nodeSelector.nodeSelectorLabel }}
```

Comment:
do you have the code for a test only? If I remember we decided to follow logic like @mur-me  implemented before there https://github.com/glifio/filecoin-chart/blob/master/templates/statefulset.yaml#L110-L113

if you are stuck, I hope @mur-me can help, if he has few mins to that

Reply:
Please review again. We fixed part of statefulset with @mur-me.